gluten-data/src/main/java/org/apache/gluten/vectorized/ArrowWritableColumnVector.java

```diff
@@ -345,6 +345,10 @@ public void retain() {
     refCnt.getAndIncrement();
   }
 
+  public void resetRefCnt(long value) {
+    refCnt.set(value);
+  }
```

Comment:
There was already a `#forceClose` API in `ColumnarBatches`, I assume that one could cover your use case.

Although let's figure out why we need to forcibly close the vectors first.